src/main/java/org/akaza/openclinica/control/core/SecureController.java

```diff
@@ -1549,6 +1550,11 @@ protected void setStudy(Study study, HttpSession session) {
                 try {
                     boardUrl = getStudyBuildService().getCurrentBoardUrl(accessToken, study);
                     study.setBoardUrl(boardUrl);
+                    try {
+                        getStudyDao().update(study);
+                    }
+                    catch (StaleStateException e) {
```

Comment:
we need to understand why this exception is thrown. It is not a good idea to catch this exception and ignore.

Reply:
@svadla-oc I've refactored the updating part to separate method that can be called independently from setting session's current study. This fixes the StaleStateException, so I removed the catch-and-ignore block on this PR's latest commit.